packages/navlink/src/types.ts

```diff
@@ -0,0 +1,15 @@
+import { IconSize } from '@fiscozen/icons/src/types'
+import { RouteLocation } from 'vue-router'
+
+export interface FzNavlinkProps<T = void> {
```

Comment:
```suggestion
export interface FzNavlinkProps<T = undefined> {
```
void è più che altro per funzioni che non ritornano

Reply:
è consigliabile usare void quando il generic è opzionale

Reply:
mi risulta di sì ma per le funzioni, comunque come vuoi è poco rilevante

Reply:
se gli dai undefined è opzionale (basta che ci sia un default in genere) ma se ad esempio poi usassimo quel T come tipo di un argomento di funzione e non passi un valore al generic la funzione richiederebbe comunque di avere un argomento di tipo undefined. Se usi void invece quell'argomento è effettivamente opzionale, un po' come se avessi fatto un overload